src/vstart.sh

```diff
@@ -458,6 +458,7 @@ prepare_conf() {
         plugin dir = $CEPH_LIB
         filestore fd cache size = 32
         run dir = $CEPH_OUT_DIR
+	crash dir = $CEPH_OUT_DIR
```

Comment:
Looks like a bad indent here

Reply:
this happens when a mix of spaces and tabs mixes with the extra +/- column in the patch.  source file is fine.